osgp/protocol-adapter-mqtt/osgp-protocol-simulator-mqtt/src/main/resources/osgp-simulator-protocol-mqtt.properties

```diff
@@ -1,3 +1,21 @@
-mqtt.simulator.spec = mqtt_simulator_peak_shaving_spec.json
+mqtt.simulator.spec=mqtt_simulator_peak_shaving_spec.json
 # Start a client publishing the messages from the JSON spec or not
-mqtt.simulator.startClient = false
+mqtt.simulator.startClient=true
+
+mqtt.simulator.broker.host=localhost
+mqtt.simulator.broker.port=1883
+
+mqtt.simulator.ssl.enabled=true
+
+mqtt.simulator.broker.ssl.port=8883
+mqtt.simulator.broker.ssl.provider=JDK
+#mqtt.simulator.broker.ssl.keystore.location=/etc/osp/keys/mqtt/mqtt_broker_keystore.jks
```

Comment:
Probably cleaner not to leave in commented out alternatives like here and in line 17.

Reply:
Updated comments